core/src/main/java/com/novoda/noplayer/PlayerView.java

```diff
@@ -19,4 +19,12 @@
 
     void setSubtitleCue(TextCues textCues);
 
+    /**
+     * whether to use the styles set by the user in the accesibility settings instead of the style
+     * that comes from the video's subtitle cues.
+     *
+     * Default: Enabled (true)
+     */
+    void setAccessibilityCaptionsStyleEnabled(boolean enabled);
```

Comment:
This doesn't look like a true javadoc. Normally it labels the params as well.

Reply:
https://github.com/novoda/no-player/pull/277/commits/746cee0bf23c52c84dce2d44f832fa643a89d6ff